pkgs/desktops/xfce4-13/default.nix

```diff
@@ -77,4 +77,21 @@ makeScope newScope (self: with self; {
   xfce4-volumed-pulse = callPackage ./xfce4-volumed-pulse { };
 
   xfwm4 = callPackage ./xfwm4 { };
+
+  inherit (xfce4-12)
+    gvfs
+    xfce4-session
+    xfce4-volumed
+    xinitrc
+    gtk-xfce-engine
+    xfce4-icon-theme
+    xfwm4-themes
+    xfce4-battery-plugin
+    xfce4-clipman-plugin
+    xfce4-embed-plugin
+    xfce4-xkb-plugin
+    xfce4-whiskermenu-plugin
+    xfce4-windowck-plugin
+    xfce4-namebar-plugin
+    xfce4-dockbarx-plugin;
```

Comment:
versions of these packages might be bumped, by copying .nix files to /xfce4-13/ directory, keeping old version
@ryantm